Update Redis to 7.2.6

stondino00 opened the pull request:
Fixes #2902

stondino00 commented:
Still same issue when testing this PR due to that possible config issue.

pachulo commented:
OK, I've done what I commented [here](https://github.com/nextcloud-snap/nextcloud-snap/pull/2859#issuecomment-2313676454); let's see if it's enough.

pachulo commented:
Would you able to test this after the build is done @scubamuc ? Thanks!

scubamuc commented:
fresh install tests completed with issues

![grafik](https://github.com/user-attachments/assets/c1f6cd2b-2e25-4867-bacf-a3095837c700)

not sure about this... shouldn't it be latest update?  #2911

![grafik](https://github.com/user-attachments/assets/be64b560-6d42-42e9-9cff-77c4a389f35a)

logs look fine

scubamuc commented:
@pachulo is this still 29.0.7 ?

pachulo commented:
> @pachulo is this still 29.0.7 ?

Yes, but it does not matter for what we want to test here.

pachulo commented:
Can you try an update when you are able to @stondino00 ? Thanks!

stondino00 commented:
Thanks SO much @pachulo! BUT I still get a cannot connect on my nextcloud instance when upgrading to this PR from the latest beta/dev track.

stondino00 commented:
3376331:C 11 Oct 2024 09:45:10.953 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
3376331:C 11 Oct 2024 09:45:10.953 * Redis version=7.2.6, bits=64, commit=00000000, modified=0, pid=3376331, just started
3376331:C 11 Oct 2024 09:45:10.953 * Configuration loaded
3376331:M 11 Oct 2024 09:45:10.953 # Failed to configure LOCALE for invalid locale name.
3376368:C 11 Oct 2024 09:45:11.193 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
3376368:C 11 Oct 2024 09:45:11.194 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
3376368:C 11 Oct 2024 09:45:11.194 * Redis version=7.2.6, bits=64, commit=00000000, modified=0, pid=3376368, just started
3376368:C 11 Oct 2024 09:45:11.194 * Configuration loaded
3376368:M 11 Oct 2024 09:45:11.194 # Failed to configure LOCALE for invalid locale name.

pachulo commented:
Try again when the build is done @stondino00

stondino00 commented:
These latest changes appear to fix the issue.  It updated just fine now.  Thanks again for your work on this!

pachulo commented:
OK, I think I've correctly done everything on 1 commit. If I remember it correctly @kyrofa once recommended us to update both Redis and the PHPRedis extension at the same time.

So, @stondino00 & @scubamuc could you both repeat your tests once this is build?

Thanks a lot!

scubamuc commented:
fresh install tests completed without issues

![grafik](https://github.com/user-attachments/assets/b4d8f8d2-a38c-4d62-bc98-79310a393677)

![grafik](https://github.com/user-attachments/assets/2a22a0be-47b9-4650-b2f2-1857753af2ef)

![grafik](https://github.com/user-attachments/assets/a3adf4cd-2e63-48fa-81cc-940a50e5bcde)

![grafik](https://github.com/user-attachments/assets/ea781e1c-9f18-4c29-bc7a-d76d1d998073)

PHP richdocuiments code error #2901, assuming this hasn't been included in nextcloud fixer yet?

![grafik](https://github.com/user-attachments/assets/af6dce63-0fb5-4150-ba5e-df2fb468c72f)

logs look fine

scubamuc commented:
@pachulo FYI

![grafik](https://github.com/user-attachments/assets/9b995e12-e8df-4c22-bc34-46e96890372a)

stondino00 commented:
Seems to update fine with no issues as far as I can see.  Great work!

pachulo commented:
> PHP richdocuiments code error https://github.com/nextcloud-snap/nextcloud-snap/issues/2901, assuming this hasn't been included in nextcloud fixer yet?

Indeed @scubamuc; nobody had time to work on it yet
